server.py

```diff
@@ -41,3 +41,26 @@ def update_product(id):
 
   """
   pass
+
+@app.route('inventory/products/<int:id>', methods=['DELETE'])
```

Comment:
Looks good!
Just one thing I'm not sure about, should we have the '/' before the `inventory/products/`?
like `/inventory/products/<int:id>`